tsl/src/continuous_aggs/options.c

```diff
@@ -173,17 +180,25 @@ update_materialized_only(ContinuousAgg *agg, bool materialized_only)
 		bool nulls[Natts_continuous_agg];
 		Datum values[Natts_continuous_agg];
 		bool repl[Natts_continuous_agg] = { false };
-		HeapTuple new;
+		bool should_free;
+		HeapTuple tuple = ExecFetchSlotHeapTuple(ti->slot, false, &should_free);
+		HeapTuple new_tuple;
+		TupleDesc tupdesc = ts_scan_iterator_tupledesc(&iterator);
```

Comment:
I've noticed that other code files use the code of `ts_scan_iterator_tupledesc` to get tuple description. It would be nice homogenise that, e.g., all relevant cases use `ts_scan_iterator_tupledesc`.